src/cli.rs

```diff
@@ -21,7 +22,7 @@ use crate::style::TableStyle;
 #[command(styles = get_styles())]
 pub struct Options {
     /// [user@]host[:port]
-    #[arg(value_parser = parse_target, value_hint = ValueHint::Hostname)]
+    #[arg(value_parser = parse_target, value_hint = ValueHint::Hostname, group = "main_action", default_value = "")]
```

Comment:
Setting `default_value = ""` allows this value to show up as required in `--help`.

Since the target and `--completions` are in the same group (`main_action`), either:

- The program is invoked with an explicit target argument value.
- The program is invoked with `--completions` and `exit(…)`s early in `main(…)` (so the blank value is never seen by the rest of the program).
- The program is invoked with with both or neither, and shows help instead of returning parsed `Options` to `main()`.